src/App.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component,lazy,Suspense } from 'react';
+import {withRouter,Route,Switch} from 'react-router-dom'
+import Blog from './containers/Blog/Blog';
+import {connect} from 'react-redux';
+const Auth = lazy(() => import('./containers/Auth/Auth'));
+const Logout = lazy(()=>import('./containers/Auth/Logout/Logout'));
+const Editpost = lazy(()=>import('./components/Editpost/Editpost'));
+
+class App extends Component {
+  
+  render(){
+    let routes=null;
```

Comment:
On refresh user session gets over can you fix this.

Reply:
Sir,done with this functionality